pkgs/development/python-modules/frida-python/default.nix

```diff
@@ -0,0 +1,39 @@
+{ lib, stdenv, fetchurl, fetchPypi, buildPythonPackage, typing-extensions }:
+let
+  version = "16.0.19";
+  devkit = fetchurl {
+    url = "https://github.com/frida/frida/releases/download/${version}/frida-core-devkit-${version}-linux-x86_64.tar.xz";
```

Comment:
Would it be possible to make this a source build? It looks like you download and unpack a prebuilt binary here.

Reply:
It would be very difficult. The frida devkit build process is extremely complex and would require a lot of work to do in a derivation. I can try to do this, but would prefer to merge like this as a first stage.

Reply:
That's Ok. I figured this was the case.
A source build doesn't need to be a blocker for this PR.